app/models/publisher_status_update.rb

```diff
@@ -30,6 +32,32 @@ class PublisherStatusUpdate < ApplicationRecord
 
   validates :publisher_id, presence: true
 
+  # After a user creates a new status then we should check to see the previous staus and call backing server
+  after_create :update_services, if: :should_update?
+
+  # Calls the promo server to update the owner state for the publisher based on the status
+  #
+  # @return [true] if the services successfully update
+  def update_services
```

Comment:
API calls like this should be done in a job. It'll automatically handle any kind of failures that occur during downtime.

Reply:
I tend to prefer system correctness over reliability in these instances, however I'll move it to a job

Reply:
Updated
https://github.com/brave-intl/publishers/pull/2002/commits/027da54e6b53f213d1e15db15d9d8381487bf6b3